### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by joyapisi**
# What I Have Done 👨🏽‍💻:
`Create the 'projects' and 'about me' section of mobile version`
-Use Figma template to create works and about me sections of the mobile version
-Use Flexbox and grid to place elements on the pages
-Use HTML and CSS linters to make my code look professional

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,46 +10,236 @@
...
+
+<!--page 2-->
+<section class="works" id="portfolio" title="works-card-1">
+  
+  <div class="card1 cards">
+    <img class="avator3" id="avator" src="sources/snap.png" alt="Avatar">
+    <div class="container">
+      <h2><b>Tonic</b></h2>
+
+     <ul class="tonic">
```
Comment (RaoAkif) [inline index.html:59]: The project card is not fully centered as per Figma Design.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -10,46 +10,236 @@
...
+    <button onclick="togglePopup4()" class="normalbutton2 seeproject">See Project</button> 
+     
+    </div>
+  </div>
+</section>
+
+<!--Page 6-->
+<article class="aboutme" id="about" title="about-me">
+
+<section class="top">
```
Comment (RaoAkif) [inline index.html:168]: Your profile overall does not seem to have a background.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image  | image  |

In this Figma Template, the About section, for example, is in a form of a card background with a rounded top-right border. Kindly look at the design again

Comment (RaoAkif) [inline index.html:168]: There is a big gap between the two sections. Kindly decrease them as per design.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image | image  |

File: index.html [hunk 3]
```diff
@@ -10,46 +10,236 @@
...
+    <li><a href="#"><img id="github"  src="sources/Vector1.png" alt="github"></a></li>        
+    <li><a href="#"><img id="peace"  src="sources/Vector3.png" alt="peace"></a></li>   
    </ul>
+  </div>  
+
+</div>
    
+<div class="action-last" title="get-my-resume-button">
+  <a class="normalbutton3"  href="#">Get my resume</a> 
+</div>
```
Comment (RaoAkif) [inline index.html:188]: The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)
Kindly put that in the `button` tag

**Comment by joyapisi** [comment]
# What I Have Done✍🏽
-Correct given revisions on my html and css files

**Pull request merged**